libbeat/esleg/eslegclient/bulkapi.go

```diff
@@ -225,3 +229,23 @@ func bulkEncode(log *logp.Logger, out BulkWriter, body []interface{}) error {
 	}
 	return nil
 }
+
+func mergeParams(m1, m2 map[string]string) map[string]string {
```

Comment:
Would be nice to have some unit tests for `mergeParams`.

Reply:
I will do it as part of the comment in the orignal PR.